Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -2156,8 +2153,13 @@ public override void Update(float elapsedClockSeconds)
                                     if (de.State != DieselEngineState.Running)
                                         de.Initialize();
                                 }
+
+                                // if train is a geared locomotive then set it to automatic operation as AI driver can't operate manual gearboxes
                                 if (de.GearBox != null)
                                     de.GearBox.GearBoxOperation = GearBoxOperation.Automatic;
+
+                                // Set gear to "low gear" at start.
+                                de.GearBox.currentGearIndex = de.GearBox.NumOfGears - 1;
```

Comment:
Appears this change is causing a new crash on diesel electric locomotives https://www.elvastower.com/forums/index.php?/topic/37993-940-stable-build-autopilot-immediate-ctd/

This line should probably be added underneath the if statement in line 2158 so that it doesn't try to set low gear if there is no gearbox.

Reply:
Ok, I have raised another PR to adjust as you have suggested.

Thanks